commit 6146c85bb2bda46d9a9e99f3682ede78b1e06347
Author: Sage Weil <sweil@redhat.com>
Date:   Thu Mar 7 11:22:27 2019 -0600

    mgr: skip shutdown and exit
    
    The python subinterpreter teardown is broken; see
    https://tracker.ceph.com/issues/38621
    
    Instead, just exit with the appropriate error code.  That way we don't
    crash and leave a core and ugly stack trace behind.
    
    Users won't care, although our leak checking will.
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/MgrStandby.cc b/src/mgr/MgrStandby.cc
index e0c80e0b00e..19c42a450b6 100644
--- a/src/mgr/MgrStandby.cc
+++ b/src/mgr/MgrStandby.cc
@@ -264,7 +264,8 @@ void MgrStandby::handle_signal(int signum)
 {
   ceph_assert(signum == SIGINT || signum == SIGTERM);
   derr << "*** Got signal " << sig_str(signum) << " ***" << dendl;
-  shutdown();
+  _exit(128 + signum);
+  //shutdown();
 }
 
 void MgrStandby::shutdown()
